fwprovider/resource_options.go

```diff
@@ -285,6 +403,24 @@ func (m *clusterOptionsModel) importFromOptionsAPI(
 		m.MigrationNetwork = types.StringNull()
 	}
 
+	if opts.NextID != nil {
+		m.NextID = &clusterOptionsNextIDModel{}
+		lower := int64(*opts.NextID.Lower)
```

Comment:
we can add something like `.PointerInt64()` to `CustomInt64` type to simplify this conversion.

Reply:
Yep, I also thought about it, but decided to solve it this way to get the implementation done.